wire-ios-data-model/Source/MLS/OneOnOne/OneOnOneMigrator.swift

```diff
@@ -171,7 +172,11 @@ public struct OneOnOneMigrator: OneOnOneMigratorInterface {
                 mlsConversation.mutableMessages.union(proteusConversation.allMessages)
             }
 
-            if !proteusConversations.isEmpty {
+            // insert system message that we moved from proteus to MLS
+            let sender = ZMUser.selfUser(in: context)
+            mlsConversation.appendMLSMigrationFinalizedSystemMessage(sender: sender, at: .now)
+
+            if !allProteusConversations.isEmpty {
```

Comment:
looking back at the history, it seems to be more accurate to update if there were any kind of proteus conversations before